app/views/news_tematica/news_tematicas/_bloque_mas_leidos.html.haml

```diff
@@ -9,5 +9,4 @@
         %h5.titulo= link_to_con_estadisticas masleido.titulo, masleido.contenido_dominio_link, @news_tematica
         %p.resumen
           = raw masleido.descripcion
-          = link_to_con_estadisticas('Leer&nbsp;más&nbsp;>>', masleido.contenido_dominio_link, @news_tematica,
-            class: 'leer_mas')
+          = link_to_con_estadisticas('Leer&nbsp;más&nbsp;>>'.html_safe, masleido.contenido_dominio_link, @news_tematica, class: 'leer_mas')
```

Comment:
I mean this, that it is not for tables...
http://www.w3schools.com/cssref/pr_text_white-space.asp

But it's okay for me as it has done @d2bit